server/tests/test_sse.py

```diff
@@ -194,3 +199,54 @@ async def test_keep_alive_real_server(minimalorg: MinimalorgRpcClients, app: Asg
                     break
 
     assert got_keepalive
+
+
+async def test_user_revoked(coolorg: CoolorgRpcClients, backend: Backend) -> None:
```

Comment:
maybe specify the name a little bit, wdyt ?
```suggestion
async def test_close_on_user_revoked(coolorg: CoolorgRpcClients, backend: Backend) -> None:
```